media-gfx/blender/blender-2.79b.recipe

```diff
@@ -190,9 +190,9 @@ BUILD()
 		-DZLIB_LIBRARY_RELEASE=`finddir B_SYSTEM_LIB_DIRECTORY`${secondaryArchSubDir}/libz.so.1 \
 		-DWITH_GHOST_SDL=ON \
 		-DWITH_BUILDINFO=OFF \
-		-DWITH_PYTHON_INSTALL=OFF \
-		-DWITH_PYTHON_INSTALL_NUMPY=OFF \
-		-DWITH_PYTHON_INSTALL_REQUESTS=OFF \
+		-DWITH_PYTHON_INSTALL=ON \
+		-DWITH_PYTHON_INSTALL_NUMPY=ON \
+		-DWITH_PYTHON_INSTALL_REQUESTS=ON \
```

Comment:
Why were these off before, but now enabled?

Reply:
There were some earlier issues with the Python 3.x dependencies which seem resolved now.